File: app/src/main/java/com/csl/studio/photoshare/app/controller/viewholder/PostViewHolder.java
package com.csl.studio.photoshare.app.model;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;

import com.csl.studio.photoshare.app.R;

public class PostViewHolder extends RecyclerView.ViewHolder {
    public ImageView _author_icon;
    public TextView _user_name;
    public TextView _post_message;
    public ImageView _post_image;
    public View _view_body;

    public PostViewHolder(View view) {
        super(view);
        _author_icon = (ImageView) view.findViewById(R.id.author_icon);
        _user_name = (TextView) view.findViewById(R.id.post_user_name);
        _post_message = (TextView) view.findViewById(R.id.post_content);
        _post_image = (ImageView) view.findViewById(R.id.post_thumbnail);
        _view_body = view;
    }

}

File: app/src/main/java/com/csl/studio/photoshare/app/controller/viewholder/PostViewHolder.kt
package com.csl.studio.photoshare.app.controller.viewholder

import android.support.v7.widget.RecyclerView
import android.view.View
import android.widget.ImageView
import android.widget.TextView

import com.csl.studio.photoshare.app.R
import kotlinx.android.synthetic.main.fragment_photo_item.view.*

class PostViewHolder(var _view_body: View) : RecyclerView.ViewHolder(_view_body) {
    var _author_icon: ImageView = _view_body.author_icon
    var _user_name: TextView = _view_body.post_user_name
    var _post_message: TextView = _view_body.post_content
    var _post_image: ImageView = _view_body.post_thumbnail

}
